conda/install.py

```diff
@@ -227,17 +227,20 @@ def exp_backoff_fn(fn, *args):
     """Mostly for retrying file operations that fail on Windows due to virus scanners"""
     if not on_win:
         return fn(*args)
-    import random
 
+    import random
     max_retries = 5
     for n in range(max_retries):
         try:
             result = fn(*args)
-        except Exception as e:
+        except EnvironmentError as e:
```

Comment:
The goal here was to have the most general error that assigns errno.  But apparently it's changed in python 3

> Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and mmap.error have been merged into OSError, and the constructor may return a subclass.

https://docs.python.org/3/library/exceptions.html#OSError.filename2

So I think `OSError` is the safe choice here?  As long as neither `errno.EPERM` or `errno.EACCES` is an `IOError` in python2.7